nodepool/files/nodepool.yml

```diff
@@ -192,6 +203,27 @@ providers:
         max-servers: 10
         availability-zones: []
         labels: *provider_pools_onmetal_labels_block
+      - name: snapshots
+        # Ignore the quota given by public cloud and only respect what we provide as max-servers
+        ignore-provider-quota: true
+        max-servers: 10
+        availability-zones: []
+        labels:
+          - name: rpc-r14-xenial_loose_artifacts-swift-7-15
+            cloud-image: rpc-r14.17.0-xenial_loose_artifacts-swift
+            flavor-name: 7
```

Comment:
Nodepool (or I think it's shade actually - can't recall) rejects a single letter flavor name, so you'll have to use ``flavor-name: "15GB Standard Instance"`` here.